ziti/cmd/ziti/cmd/edge/login.go

```diff
@@ -186,17 +187,17 @@ func (o *loginOptions) Run() error {
 		o.Cert = absCertPath
 	}
 
-	loginIdentity := &util.RestClientIdentity{
+	loginIdentity := &util.RestClientEdgeIdentity{
 		Url:       host,
 		Username:  o.Username,
 		Token:     o.Token,
 		LoginTime: time.Now().Format(time.RFC3339),
-		Cert:      o.Cert,
+		CaCert:    o.Cert,
```

Comment:
this change sticks out to me - either o.Cert should be o.CaCert or this should not be CaCert?

Reply:
Switched o.Cert to o.CaCert. The edge identity only has one cert, whereas the fabric identity has the signed client cert plus the ca cert. So since I had to distinguish them, made sense to make it more consistent